gateway/1.11/modules/ROOT/pages/flex-agent-policies.adoc

```diff
@@ -43,6 +43,7 @@ Both A2A and MCP support the following policies for Server-Sent Events (SSE) str
 |===
 |Policy |Summary
 | xref:policies-included-mcp-attribute-access-control.adoc[MCP Attribute-Based Access Control] | Controls access to tools, resources, and prompts based on user information such as Tiers, IP, Headers, or Claims
+| xref:policies-included-mcp-pii-detector.adoc[MCP PII Detector] | Blocks elicitation responses containing personally identifiable information (PII) from reaching MCP servers
```

Comment:
This isn't a big deal and you can address it some other time, but in tables, if you use periods in one row, you need to use them in all the rows.

Reply:
You're right, going to ignore it because we're EOLing this policy next week.